packages/playground/src/twoslashInlays.ts

```diff
@@ -32,7 +32,7 @@ export const createTwoslashInlayProvider = (sandbox: Sandbox) => {
         if (!hint || !hint.displayParts) continue
 
         // Make a one-liner
-        let text = hint.displayParts.map(d => d.text).join("").replace(/\r?\n/g, "").replace(/  /g, "")
+        let text = hint.displayParts.map(d => d.text).join("").replace(/\s+/g, " ")
```

Comment:
I'm not a fan of this, but the reality is that this code already does "the wrong thing" by nature of touching the contents of the tooltip. It's totally possible to have a string literal type which consists of multiple spaces and yet the old code would futz with it.

The old code (`/  /g, ""`) would also do very odd things like `" " -> " "`, but `"  " -> ""` and then `"   " -> " "`.

Reply:
Maybe replacing `/\r?\n\s*` would be better? Assuming you’re just trying to get rid of leading indentation in multi-line object types

Reply:
That's a great idea!